x-pack/legacy/plugins/siem/public/pages/detection_engine/create_rule/components/mitre/index.tsx

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButtonEmpty,
+  EuiButtonIcon,
+  EuiFormRow,
+  EuiSelect,
+  EuiSpacer,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiComboBox,
+  EuiFormControlLayout,
+} from '@elastic/eui';
+import { isEmpty, kebabCase, camelCase } from 'lodash/fp';
+import React, { ChangeEvent, useCallback, useMemo, useState } from 'react';
```

Comment:
Doesn't look like `useMemo`, `useState` are used anymore:


```suggestion
import React, { ChangeEvent, useCallback } from 'react';
```